README.md

```diff
@@ -150,15 +150,16 @@ http_request_duration_seconds_sum{handler="metrics",method="get"} 0.021113919999
 http_request_duration_seconds_count{handler="metrics",method="get"} 30
 ```
 
-kube-state-metrics exposes the build info metric:
+kube-state-metrics also exposes build and configuration metrics:
 ```
 kube_state_metrics_build_info{branch="master",goversion="go1.15.3",revision="6c9d775d",version="v2.0.0-beta"} 1
+kube_state_metrics_shard_ordinal 0
```

Comment:
Do you mind including a couple of use cases for these metrics, similar to how the `kube_state_metrics_build_info ` has below. Thanks!

(So either any alert suggestions or when users should use this)

Reply:
Good point, added